src/main/java/org/icgc/argo/program_service/services/ego/model/entity/EgoUser.java

```diff
@@ -2,13 +2,15 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.Accessors;
 import org.icgc.argo.program_service.proto.UserRole;
 import java.util.UUID;
 
 @Accessors(chain = true)
+@Builder
```

Comment:
With `@Accessors(chain=true)` it achieves the same result as `@Builder` when used with `@Data`. Since the object is mutable, there is only really a cosmetic difference. For the `@Accessors` approach the object would be built by:
```java
new EgoUser().setId("someid").setFirstName("Jon");
```
where as for the `@Builder` approach the object would be built by:
```java
EgoUser.builder().id("someId").firstName("Jon").build();
```

Also, refer to [Do not use @Builder when using @AfterMapping or @BeforeMapping](https://github.com/icgc-argo/program-service/blob/master/CONTRIBUTING.md#1-do-not-use-builder-when-using-aftermapping-or-beforemapping).